src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -137,6 +137,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DocumentationFile />
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(PublishDocumentationFile)' == ''">
+    <PublishDocumentationFile Condition="'$(GenerateDocumentationFile)' == 'true'">true</PublishDocumentationFile>
```

Comment:
> think we want this new opt-out to disable that behavior for project refs too.

I think so. I'm thinking about renaming it to `PublishDocumentationFiles` (plural) to:
* Control adding `@(FinalDocFile)` to `@(ResolvedFileToPublish)`
* Filter out xml files returned from the `ResolvePublishAssemblies` task (maybe add a task parameter although it doesn't really seem to fit in there).

Thoughts?